controller/src/main/java/io/pravega/controller/store/host/ZKHostStore.java

```diff
@@ -53,15 +55,19 @@
 
         zkClient = client;
         zkPath = ZKPaths.makePath("cluster", "segmentContainerHostMapping");
+        hostContainerMapNode = new NodeCache(zkClient, zkPath);
+
         segmentMapper = new SegmentToContainerMapper(containerCount);
     }
 
     //Ensure required zk node is present in zookeeper.
     @Synchronized
+    @SneakyThrows(Exception.class)
```

Comment:
Can we narrow this down to `IOException.class`? I can't tell from this view where the exception might be coming from.

Reply:
Curator has signature for NodeCache.start() as throws Exception